deployment/ccip/changeset/testhelpers/test_helpers_solana_v0_1_0.go

```diff
@@ -1540,6 +1490,9 @@ func AddLaneWithDefaultPricesAndFeeQuoterConfig(t *testing.T, e *DeployedEnv, st
 		aptosState := state.AptosChains[from]
 		tokenPrices[aptosState.LinkTokenAddress.StringLong()] = deployment.EDecMult(20, 28)
 		tokenPrices[shared.AptosAPTAddress] = deployment.EDecMult(5, 28)
+	case chainsel.FamilyTon:
+		tonState := state.TonChains[from]
+		tokenPrices[tonState.LinkTokenAddress.String()] = deployment.EDecMult(20, 28)
```

Comment:
Need to double check this, LINK will have 9 decimals on TON like on Solana (not 18)